### DISCUSSION TITLE
Link to file after uploading

### DISCUSSION THREAD

**Discussion by Milifaru** [rejetto/hfs#803] [Q&A]
I apologize if the question is stupid.

Can you help, perhaps suggesting a script so that after uploading a file a modal window appears with a link to the uploaded file and a "Copy link" button.

I created a folder in which you can't get a list of files, but you can upload and download them. This is necessary so that the file can only be accessed via a link, and this file cannot be found in any other way.

In general, by uploading a file and adding its name to the folder link, you can manually get the desired link. But this is not convenient and if a file with this name already exists, another one is created with a serial number, but this is not visible to the one uploading the file.

**Comment by rejetto** [comment]
this is an interesting use case, and your problem is quite real.
this should be solved by hfs itself, without any script.
i will solve it in version 0.55, as 0.54 is already closing.

**Comment by rejetto** [comment]
i will keep you posted here for when the feature is available.
i hope it will be within 2 weeks.

**Reply by Milifaru** [reply]
Thank you very much for your answer, I will follow the news.

**Comment by rejetto** [comment]
in next release
<img width="417" alt="image" src="https://github.com/user-attachments/assets/9ebebdfb-e072-46bc-bf75-5ad1259cbd5f">

**Comment by Milifaru** [comment]
I received the update, great job, thanks a lot!
